# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux24benh/82-raid5-niceer-output.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


I also sent this patch to the kernel hackers - because the output is
really confusing when the software RAID code does not choose the fastest
mode ... Rene

--- linuxi.orig/drivers/md/xor.c	Sun Sep 30 21:26:06 2001
+++ linux/drivers/md/xor.c	Fri Dec  7 18:16:10 2001
@@ -127,6 +127,8 @@
 
 #ifdef XOR_SELECT_TEMPLATE
 	fastest = XOR_SELECT_TEMPLATE(fastest);
+	printk("raid5: using %s to write around L2 cache\n",
+		fastest->name);
 #endif
 
 	active_template = fastest;
